sdk/keyvault/keyvault-keys/samples-dev/cryptography.ts

```diff
@@ -18,7 +18,7 @@ export async function main(): Promise<void> {
   // This sample uses DefaultAzureCredential, which supports a number of authentication mechanisms.
```

Comment:
This is kinda weird, and can ben confusing. Maybe we remove that comment anywhere we have it but not sure